### PR TITLE
Add an new cat

### PR DESCRIPTION
**Pull request by guinetn**
Use the 'Add' button or press [Enter]

### CODE REVIEW

File: src/components/CatList.html [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 <ul>
   {{#each cats as cat}}
     <Cat cat='{{ cat }}'></Cat>
   {{/each}}
 </ul>
+<p>Count: {{count}}</p>
 
 <script>
-  import Cat from './Cat.html';
+  import Cat from '../components/Cat.html';
```
Comment (charpeni) [inline src/components/CatList.html:10]: Was there an issue with this import ?

**Review by charpeni — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Unfortunately, it does not seem to work properly. I can add a new cat to the list, but it seems rendered many cycles after and I got this error :

`main.js:280 Uncaught TypeError: cat.set is not a function(…)`

File: src/components/CatList.html (hunk 1 above)
Comment (guinetn) [reply src/components/CatList.html:10]: Hi, I didn't get any error. Wich browser are you using?
I publish it here: https://github.com/guinetn/svelte-example

This has no impact but first is better:
-  import Cat from './Cat.html';
+  import Cat from '../components/Cat.html';

Comment (charpeni) [reply src/components/CatList.html:10]: @guinetn On chrome, latest version.

**Review by charpeni — DISMISSED** [review]

**Review by charpeni — APPROVED** [review]

**Pull request merged**